pkg/agent/agent.go

```diff
@@ -57,6 +75,9 @@ func NewAgent(ctx context.Context,
 	managers = append(managers, analyzerManager)
 	avoidanceManager := executor.NewActionExecutor(kubeClient, nodeName, podInformer, nodeInformer, noticeCh, runtimeEndpoint)
 	managers = append(managers, avoidanceManager)
+	cpuManager := cm.NewAdvancedCpuManager(kubeClient, nodeName, podInformer, nodeInformer, runtimeEndpoint, cadvisorManager)
+	managers = append(managers, cpuManager)
+	metrics.RegisterResourceMetrics(agent.host, utils.NewCpuStateProvider(cadvisorManager, podInformer.Lister(), useBt, cpuManager.GetExclusiveCpu))
```

Comment:
After CpuStateProvider merge into collector, we could use data from collecter here.

Reply:
Yes~